src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$text-main-color: #060b35;
```

Comment:
As per BEM convention util files should be named starting with '_'
hear it should be  '_variables.scss'